language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -130,18 +130,28 @@ packageNameText pkgId mbPkgIdent = maybe (unPackageId pkgId) unPackageName mbPkg
 
 newtype Scope = Scope {unscope :: String}
 newtype Dependency = Dependency {undependency :: T.Text}  deriving (Eq, Ord)
+newtype SdkVersion = SdkVersion String
```

Comment:
No need to introduce your own newtype, you can use `DA.Daml.Project.Types.SdkVersion`.

Reply:
Ok!